components/exercises/chapter-5-oscar/Everything/some.test.ts

```diff
@@ -0,0 +1,65 @@
+import { some } from './some';
+
+describe('some', () => {
+  it('should return true if each element satisfies the condition for numbers', () => {
+    const arr = [2, 4, 6, 8];
+    const callBack = (num: number) => num < 10;
+    expect(some(arr, callBack)).toBe(true);
+  });
+
+  it('should return false if at least one element does not satisfy the condition for numbers', () => {
+    const arr = [20, 4, 7, 8];
+    const callBack = (num: number) => num < 10;
+    expect(some(arr, callBack)).toBe(false);
+  });
```

Comment:
Nope, `some` should return true **if at least one element satisfies the condition**

Let's update the function code so this test passes
```js
    const arr = [20, 4, 7, 8];
    const callBack = (num: number) => num < 10;
    expect(some(arr, callBack)).toBe(true);
```

Reply:
Actually forget about this, read my comment below